usr.bin/ctags/fortran.c

```diff
@@ -48,7 +48,7 @@ PF_funcs(void)
 	char	*cp;
 	char	tok[MAXTOKEN];
 
-	for (pfcnt = NO;;) {
+	for (pfcnt = false;;) {
```

Comment:
Should `int PF_funcs(void)` be changed to `bool PF_funcs(void)` due to these changes?

Reply:
I think so because we're return pfcnt 3 lines below.

Reply:
Thanks for checking, you are right. While fixing it I also noticed that "skip_key" in C.c and ctags.h should return bool. I'll amend those changes to the commit.